flows/definition/testdata/TestChangeLanguage_change_language_to_spa.snap

```diff
@@ -63,43 +63,6 @@
                     "Which pill?"
                 ]
             }
-        },
```

Comment:
this was a mistake - it was leaving behind the translation that becomes the flow text